cargo-shuttle/README.md

```diff
@@ -0,0 +1,210 @@
+
+OPTIONS:
+        --api-url <api-url>
+            Run this command against the api at the supplied url [env: SHUTTLE_API=]
+
+        --name <name>                              Specify the name of the project (overrides crate name)
+        --working-directory <working-directory>    Specify the working directory [default: .]
+
+SUBCOMMANDS:
+    auth      create user credentials for the shuttle platform
+    delete    delete the latest deployment for a shuttle project
+    deploy    deploy a shuttle project
+    help      Prints this message or the help of the given subcommand(s)
+    init      create a new shuttle project
+    login     login to the shuttle platform
+    logs      view the logs of a shuttle project
+    run       run a shuttle project locally
+    status    view the status of a shuttle project
+```
+
+### Subcommand: `init`
+
+To initialize a shuttle project with boilerplates, run `cargo shuttle init [OPTIONS] [PATH]`. 
+
+Currently, `cargo shuttle init` supports the following options include:
+
+- `--axum`: for [axum](https://github.com/tokio-rs/axum) framework
+- `--poem`: for [poem](https://github.com/poem-web/poem) framework
+- `--rocket`: for [rocket](https://rocket.rs/) framework
+- `--tide`: for [tide](https://github.com/http-rs/tide) framework
+- `--tower`: for [tower](https://github.com/tower-rs/tower) library
+
+For example, running the following command will initialize a project for [rocket](https://rocket.rs/):
+
+```sh
+$ cargo shuttle init my-rocket-app
+```
+
+This should generate the following dependency in `Cargo.toml`:
+```toml
+shuttle-service = { version = "0.4.0", features = ["web-rocket"] }
+```
+
+The following boilerplate code should be generated into `src/lib.rs`:
+
+```rust
+#[macro_use]
+extern crate rocket;
+
+use shuttle_service::ShuttleRocket;
+
+#[get("/hello")]
+fn hello() -> &'static str {
+    "Hello, world!"
+}
+
+#[shuttle_service::main]
+async fn init() -> ShuttleRocket {
+    let rocket = rocket::build().mount("/", routes![hello]);
+
+    Ok(rocket)
+}
+```
+
+### Subcommand: `run`
+
+To run the shuttle project locally, use the following command:
+
+```sh
+# Inside your shuttle project
+$ cargo shuttle run
+```
+
+This will compile your shuttle project and start it on the default port `8000`. Test it by:
+
+```sh
+$ curl http://localhost:8000/hello
+Hello, world!
+```
+
+### Subcommand: `login`
+
+Use `cargo shuttle login` inside your shuttle project to generate an API key for the shuttle platform:
+
+```sh
+# Inside a shuttle project
+$ cargo shuttle login
+```
+
+This should automatically open a browser window with an auto-generated API key for your project. Simply copy-paste the API key back in your terminal or run the following command to complete login:
+
+```sh
+$ cargo shuttle login --api-key your-api-key-from-browser
+```
+
+### Subcommand: `deploy`
+
+To deploy your shuttle project to the cloud, run:
+
+```sh
+$ $ cargo shuttle deploy
```

Comment:
Double '$' :)

Reply:
I should not submit PRs late in the evening haha :D